app/controllers/vaccinations_controller.rb

```diff
@@ -8,6 +8,7 @@ class VaccinationsController < ApplicationController
   before_action :set_vaccination_record, only: %i[show confirm record]
   before_action :set_consent_response, only: %i[show confirm]
   before_action :set_triage, only: %i[show confirm]
+  before_action :set_draft_consent_response, only: %i[show consent]
```

Comment:
Does `consent` need this? I don't see it using the `@draft_consent_response` in the controller and it appears to always redirect?